test/dlc_tests/benchmark/sagemaker/mxnet/training/resources/scripts/smtrain-resnet50-imagenet.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# PreRequisities
+# Install mxnet >=1.3b and gluoncv on each machine
+# Ensure data is kept at ~/data for each machine or change the data paths below
+# Example command to start the training job
+# Specify hosts in the file `hosts`
+
+set -ex
+# p3 instances have larger GPU memory, so a higher batch size can be used
+# GPU_MEM=`nvidia-smi --query-gpu=memory.total --format=csv,noheader -i 0 | awk '{print $1}'`
+# returns per GPU memory in MB
+# if [ $GPU_MEM -gt 15000 ] ; then BATCH_SIZE=1280; else BATCH_SIZE=256; fi
+
+# Training
+
+python mxnet_imagenet_resnet50.py --num-epochs "40"
```

Comment:
Please add a comment here explaining what `--num-epochs` should be set to if this test needs to be run in the PR context vs an actual perf test run.

Please also add a comment to the pytest test function which points to this comment and this line of code in this file.

Reply:
Added comments at both the mentioned places.